util/avro/export.py

```diff
@@ -60,23 +61,47 @@ def create_node_dict(node_id, node_name, values, node_schema, edges):
     return node_dict
 
 
+def split_by_n(input_list, n=1000):
+    return [input_list[x:x + n] for x in range(0, len(input_list), n)]
+
+
+def get_ids_from_table(db, table, ids, id_column):
+    data = None
+
+    for ids_chunk in split_by_n(ids):
+        current_chunk_data = db \
+            .option("query", "SELECT * FROM {} WHERE {} IN ('{}')".format(table, id_column, "','".join(ids_chunk))) \
+            .load()
+
+        if data:
+            data = data.union(current_chunk_data)
+        else:
+            data = current_chunk_data
+
+    return data if data else None
+
+
 def export_avro(
-    spark,
-    schema,
-    metadata,
-    dd_tables,
-    traverse_order,
-    case_ids,
-    db_url,
-    db_user,
-    db_pass,
+        spark,
+        schema,
+        metadata,
+        dd_tables,
+        traverse_order,
+        case_ids,
+        db_url,
+        db_user,
+        db_pass,
 ):
     node_label, edge_label = dd_tables
 
     db = spark.read.format("jdbc").options(
         url=db_url, user=db_user, password=db_pass, driver="org.postgresql.Driver"
     )
 
+    db_tables = spark.read.format("jdbc").options(
```

Comment:
db == db_table. Why do we need both?

Reply:
Somehow Spark is caching `options` inside the object, so I can't reuse it directly.